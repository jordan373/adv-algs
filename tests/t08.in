23
addEdge 0   1   100
addEdge 0   2   68
addEdge 0   3   53
addEdge 1   2   32
addEdge 1   3   71
addEdge 2   3   82
addEdge 3   4   41
addEdge 4   5   28
addEdge 4   6   26
addEdge 4   8   29
addEdge 4   9   29
addEdge 4   10  33
addEdge 5   6   88
addEdge 6   7   62
addEdge 8   9   64
addEdge 10  11  93
addEdge 10  12  78
addEdge 11  13  72
addEdge 12  13  76
addEdge 13  14  17
addEdge 14  15  66
addEdge 14  18  97
addEdge 15  16  4
addEdge 15  17  54
addEdge 15  18  40
addEdge 16  17  8
addEdge 16  18  85
addEdge 18  19  19
addEdge 18  20  71
addEdge 20  21  45
addEdge 20  22  30
addEdge 21  22  25
kruskal
printDistance
printParents
