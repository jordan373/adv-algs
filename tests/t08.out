Running the Kruskal's algorithm.
MST is:
3 2 
2 
1 0 
4 0 
6 5 9 8 10 3 
4 
4 7 
6 
4 
4 
4 12 
13 
13 10 
14 11 12 
13 15 
16 18 14 
15 17 
16 
19 15 20 
18 
22 18 
22 
21 20 
Weight of MST is: 907
Distances:
0 0 2 0 0 0 1 0 0 0 0 0 0 0 1 0 1 0 0 3 0 0 1 
Parents:
6 2 19 2 6 6 2 6 6 6 2 19 19 19 19 19 19 19 19 19 22 22 19 
